Marlin/src/core/serial.cpp

```diff
@@ -29,7 +29,7 @@ static const char errormagic[] PROGMEM = "Error:";
 static const char echomagic[]  PROGMEM = "echo:";
 
 #if NUM_SERIAL > 1
-  int8_t serial_port_index = SERIAL_PORT;
+  int8_t serial_port_index = 0;
```

Comment:
This will be the default serial port for all output. 0 is `SERIAL_PORT`, 1 is `SERIAL_PORT_2` and 0x7F will cause all general output to go to both serial ports.